packages/sdk/src/react/queries/highestOffer.ts

```diff
@@ -0,0 +1,65 @@
+import { queryOptions } from '@tanstack/react-query';
+import type { Order, SdkConfig } from '../../types';
+import { collectableKeys, getMarketplaceClient } from '../_internal';
+import type { UseQueryParameters } from 'wagmi/query';
+
+export type UseHighestOfferArgs = {
+	collectionAddress: string;
+	tokenId: string;
+	chainId: number;
+	query?: UseQueryParameters;
+};
+
+// TODO: Take this to a more global place
+export type BigIntOrder = Omit<Order, 'priceAmount' | 'priceAmountNet'> & {
+	priceAmount: bigint;
+	priceAmountNet: bigint;
+};
+
+/**
+ * Fetches the highest offer for a specific collectible
+ *
+ * @param args - Arguments for the API call
+ * @param config - SDK configuration
+ * @returns The highest offer data with BigInt conversion
```

Comment:
```suggestion
 * @returns The highest offer
```
Sorry I overlooked this on first review, I dont think we need more text here